uploadNeuroDB/minc_insertion.pl

```diff
@@ -145,44 +145,44 @@
 &Getopt::Tabular::SetHelp($Help, $Usage);
 &Getopt::Tabular::GetOptions(\@opt_table, \@ARGV) || exit 1;
 
-
-# input option error checking
-{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
-
-
-if ($profile && !@Settings::db) { 
-    print "\n\tERROR: You don't have a ".
-    "configuration file named '$profile' in:  $ENV{LORIS_CONFIG}/.loris_mri/ \n\n";
-    exit 2; 
+if(!$ENV{LORIS_CONFIG}) {
+	print STDERR "\n\tERROR: Environment variable 'LORIS_CONFIG' not set\n\n";
+	exit 1;  # TODO replace by appropriate exit code from ExitCodes once it is defined
 }
 
-if (!$profile) { 
+if (!defined$profile || !-e "$ENV{LORIS_CONFIG}/.loris_mri/$profile") {
```

Comment:
do we need a space between `defined` and `$profile`?
just out of curiosity, I would tend to put `()` around `$profile` because it makes it more readable to me, but is this not needed or not desirable?

Reply:
I usually put a space after `defined` but I guess I forgot to put one this time. I am actually surprised that this even works! But I tested again and this is valid. Adding parentheses around the argument is also valid and is entirely a matter of programming style. I'll add a space.